spotbugs/src/main/java/edu/umd/cs/findbugs/TextUICommandLine.java

```diff
@@ -816,4 +818,19 @@ private void handleAnalyzeFromFile(String filePath) throws IOException {
     private UserPreferences getUserPreferences() {
         return project.getConfiguration();
     }
+
+    /**
+     * Adds a reporter to the aggregating list of reporters, assuming it does not duplicate a known reporter.
```

Comment:
I think the "assuming" bit could be rephrased, because it is ignoring the reporter in case it is deemed to be a duplicate

Reply:
reworded